app/src/main/java/mozilla/lockbox/presenter/AuthPresenter.kt

```diff
@@ -95,4 +103,18 @@ class AuthPresenter(
             dispatcher.dispatch(DataStoreAction.Unlock)
         }
     }
+
+    private fun finishResponse(passwords: List<ServerPassword>? = null) {
+        if (passwords != null) {
+            val response =
+                responseBuilder.buildFilteredFillResponse(view.context, passwords)
+                    ?: responseBuilder.buildFallbackFillResponse(view.context)
+            // This should send off to the searchable list
+            // https://github.com/mozilla-lockbox/lockbox-android/issues/421
+
+            view.setFillResponseAndFinish(response)
+        } else {
+            view.setFillResponseAndFinish(null)
```

Comment:
Why do we want the fill response to be null here and not `responseBuilder.buildFallbackFillResponse`?

Reply:
fallback sets up a "Search ..." prompt while `null` sets up a "Unlock ..." prompt.